src/Service/LeantimeApiService.php

```diff
@@ -378,6 +377,7 @@ public function getPlanningDataWeeks(): PlanningData
         $weeks = $planning->weeks;
 
         $currentYear = (int) (new \DateTime())->format('Y');
+        $currentWeek = (int) (new \DateTime())->format('W');
 
         for ($weekNumber = 1; $weekNumber <= 52; ++$weekNumber) {
```

Comment:
A year can actually have 53 weeks (cf. https://en.wikipedia.org/wiki/ISO_week_date#Weeks_per_year), and the year 2026 (in the near future is one of them).

Reply:
I can't take credit for this. So, I have pasted your suggestion into a comment.